CHANGELOG.rst

```diff
@@ -8,6 +8,13 @@ Changelog
 
  .. note:: This version is not yet released and is under active development.
 
+* Added support for
+  :class:`~cryptography.hazmat.primitives.hashes.SM3` and
+  :class:`~cryptography.hazmat.primitives.ciphers.algorithms.SM4`,
+  when using OpenSSL 1.1.1. These algorithms are provided for compatibility
+  and compliance in regions where they may be required, and are not generally
```

Comment:
Let's drop the word compliance and only say compatibility.

The genocidal, authoritarian, Chinese government is not something anyone should comply with. Compatibility is morally neutral and strictly factual.